code/modules/clothing/gloves/color.dm

```diff
@@ -20,6 +19,15 @@
 		slapcraft_recipes = slapcraft_recipe_list,\
 	)
 
+/obj/item/clothing/gloves/color/black/security
+	name = "security gloves"
+	desc = "These security gloves come with microchips that help the user quickly restrain suspects."
+	icon_state = "sec"
+	clothing_traits = list(TRAIT_FAST_CUFFING)
+
+/obj/item/clothing/gloves/color/black/security/blu
+	icon_state = "sec_blu"
```

Comment:
why not `blue`

Reply:
tf2

Reply:
Blutarch Mann's legacy lives on in these shitty themed gloves.

Reply:
silly but fine